tests/end2end/cypress/integration/form_edition_value_relation_field-ghaction.js

```diff
@@ -1,44 +1,48 @@
 describe('Form edition all field type', function() {
-    before(function(){
+    beforeEach(function(){
         // Runs before each tests in the block
         cy.visit('/index.php/view/map/?repository=testsrepository&project=form_edition_value_relation_field')
         // Start by launching feature form
         cy.get('#button-edition').click()
         cy.get('#edition-draw').click()
     })
 
-    it('No expression menulist 4 values + 1 empty value', function () {
-        cy.get('#jforms_view_edition_code_without_exp option').should('have.length', 5)
-        cy.get('#jforms_view_edition_code_without_exp option').first().should('have.text', '')
-            .nextAll().each(($opt, index, $options) => {
-                cy.wrap($opt).should('not.have.text', '')
-            })
-    })
+    it('Check initial states', function() {
+        context('No expression menulist 4 values + 1 empty value', function () {
+            cy.get('#jforms_view_edition_code_without_exp option').should('have.length', 5)
+            cy.get('#jforms_view_edition_code_without_exp option').first().should('have.text', '')
+                .nextAll().each(($opt, index, $options) => {
+                    cy.wrap($opt).should('not.have.text', '')
+                })
+        })
 
-    it('Simple expression menulist 2 values + 1 empty value', function () {
-        cy.get('#jforms_view_edition_code_with_simple_exp option').should('have.length', 3)
-        cy.get('#jforms_view_edition_code_with_simple_exp option').first().should('have.text', '')
-            .nextAll().each(($opt, index, $options) => {
-                cy.wrap($opt).should('not.have.text', '')
-            })
-    })
+        context('Simple expression menulist 2 values + 1 empty value', function () {
```

Comment:
@rldhont why `context`? It is written in link above that `context` is like `describe` but there are not several tests in your `context`s.

https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests#Test-Structure

Reply:
@rldhont gentle ping